contracts/utils/OperatorSignedHashes.sol

```diff
@@ -55,37 +56,25 @@ contract OperatorSignedHashes {
     // Register agents type hash
     bytes32 public constant REGISTER_AGENTS_TYPE_HASH =
         keccak256("RegisterAgents(address operator,address serviceOwner,uint256 serviceId,bytes32 agentsData,uint256 nonce)");
-    // Original domain separator value
-    bytes32 public immutable domainSeparator;
-    // Original chain Id
-    uint256 public immutable chainId;
```

Comment:
Not needed as constants anymore as domain separator is calculated each time now.